components/feature/media/src/main/java/mozilla/components/feature/media/service/MediaSessionServiceDelegate.kt

```diff
@@ -116,7 +125,13 @@ internal class MediaSessionServiceDelegate(
         }
 
         updateMediaSession(state)
-        updateNotification(state)
+        MainScope().launch() {
+            updateNotification(state)
+        }
+    }
+
+    private fun getNotificationDispatcher(): CoroutineDispatcher {
```

Comment:
Is this still getting used in the current PR? I can't find where we are calling this?

Reply:
No usage of this anymore.  I'll remove